### PR TITLE
Added Python Book Resources

### PR DESCRIPTION
**Pull request by BrianHHough**
I added 5 free, substantial books on Python from the web to this large repository of resources, with a primary focus on machine learning and AI.

## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Improve Repo

### Description 
Added 5 substantial free books on learning Python, with a primary focus on machine learning and AI.

### Why is this valuable (or not)?
New users can learn how to build ML models and build algorithms with Python.

### How do we know it's really free?
The books were sourced from various online locations/webpages, and were checked before including.

### For book lists, is it a book? For course lists, is it a course? etc.
They are PDF books online.

### Checklist:
- [X] Not a duplicate
- [X] Included author(s) if appropriate
- [X] Lists are in alphabetical order
- [X] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by SethiPandi** [comment]
Hey, and welcome to free programming books! :wave:

You've actually added these to the wrong file, please see `free-programming-books.md`

The file has a link to it and states:
> English (The list of books in English is here for historical reasons.)

I'm not 100% sure how appropriate this PR is, as it's to a LinkedIn post that links / attaches lots of resources it seems, it would probably be better to add links to the original source of those, but regardless it'd be good to at least get these in the correct file for now.

For example, the first link (`https://www.linkedin.com/posts/vipulppatel_selective-overview-of-deep-learning-activity-6599895280643002368-tY9N`) can easily be obtained from its original source here:
https://arxiv.org/abs/1904.05526

**Comment by BrianHHough** [comment]
Hi @SethiPandi — thanks for the comments and warm welcome to Hacktoberfest! 🎃 I have made the following changes based on your feedback:
- I moved my additions to the correct file `free-programming-books.md`
- I removed my original additions from `free-programming-books-en.md`
- I accounted for duplicates in the correct file.
- I put in the correct links so they are non-LinkedIn and link to the original source.

I have edited 2 files:
1.) `free-programming-books-en.md`
2.) `free-programming-books.md`

A link to my corrected code is here: https://github.com/EbookFoundation/free-programming-books/pull/4866/commits/dd34136c7b961cafc68602e49c33021191bf04f9

Please let me know if there's anything more you need from me to edit! 😊

File: free-programming-books.md [hunk 1]
```diff
@@ -2392,6 +2395,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [The Programming Historian](http://niche-canada.org/research/niche-digital-infrastructure-project/the-programming-historian/) - William J. Turkel, Adam Crymble and Alan MacEachern
 * [The Python GTK+ 3 Tutorial](http://python-gtk-3-tutorial.readthedocs.org/en/latest/)
 * [The Standard Python Library](http://effbot.org/librarybook/) - Fredrik Lundh
+* [The Ultimate Guide to 12 Dimensionality Reduction Techniques (with Python codes)](https://www.analyticsvidhya.com/blog/2018/08/dimensionality-reduction-techniques-python/) - Pulkit Sharma
```
Comment (eshellman) [inline free-programming-books.md:2398]: this belongs in data mining or math for computer science

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
also, the description for the PR and title could be more accurate and helpful

**Comment by eshellman** [comment]
be sure to check the output of Travis-CI for linter errors

**Comment by BrianHHough** [comment]
> be sure to check the output of Travis-CI for linter errors

Hi @eshellman ! Apologies — I am not sure why I was getting that Travis CI `build failed` error...however, I have incorporated all of your changes/feedback/suggestions into a new PR and will close this one. The one incorporating all of your feedback with a clean/passing Travis CI build is here: https://github.com/EbookFoundation/free-programming-books/pull/4882.

I'll close this one, given the one linked above seems to be the best version now ^

**Pull request closed without merging**